ocrd_utils/ocrd_utils/__init__.py

```diff
@@ -809,3 +811,26 @@ def parse_json_string_or_file(value='{}'):    # pylint: disable=unused-argument
     if err:
         raise err       # pylint: disable=raising-bad-type
     return ret
+
+def assert_file_grp_cardinality(grps, n):
+    """
+    Assert that a string of comma-separated fileGrps contains exactly ``n`` entries.
+    """
+    if isinstance(grps, str):
+        grps = grps.split(',')
+    assert len(grps) == n, \
+            "Expected exactly %d output file group%s, but '%s' has %d" % (
+                n, '' if n == 1 else 's', grps, len(grps))
```

Comment:
Since I'm not a language person: Is "cardinality" correct, should it not be "arity"?

Reply:
> Since I'm not a language person: Is "cardinality" correct, should it not be "arity"?

I'd say _cardinality_ (of the set of file groups) is the general concept, _arity_ is the cardinality of some operator, so both terms apply in this case.